public/javascripts/lists.js

```diff
@@ -46,23 +45,28 @@ document.addEventListener('DOMContentLoaded', async (event) => {
 
                     const updateBtn = document.querySelector('.task-edit-update-button');
                     updateBtn.addEventListener('click', async (e) => {
+                        e.preventDefault();
                         const titleValue = document.getElementById("task-name-edit").value;
                         const descriptionValue = document.getElementById("task-description-edit").value;
                         const dateValue = document.getElementById("task-date-edit").value;
-                        const timeValue = document.getElementById("task-time-edit").value;
+                        const timeValue = document.getElementById("task-time-edit").value || null;
                         const experienceValue = document.getElementById("task-exp-edit").value;
 
                         const res = await fetch(`/tasks/${elem.id}`, {
                             method: "PUT",
                             headers: {
                                 "Content-Type": "application/json"
                             },
-                            body: {
-                                
-
```

Comment:
This will remove update functionality for tasks. We need to add a body into the request.